### ISSUE TITLE
DISCUSSION: Version PHP file is not optimal

### ISSUE THREAD

**Issue by tmccormi** [LibreHealthIO/LibreEHR#1237]
The old method of tracking version / release information is not being used in any real sense of the word.
The intended model was to allow detection of "upgrade to database needed" as well as "tested for production".

We aren't using the database upgrade detection at all and we haven't done a 'production' release in a long time.   

The questions are: 

- Do we need this in it current format? 
- Are there better ways to do this, from commit data or TAGS consider the new auto-updater?
- Should we use automatic migration scripts (like laravel)? Instead of manual sqlupgrade.php?

version.php
<?php
// Software version identification.
// This is used for display purposes, and also the major/minor/patch
// numbers are stored in the database and used to determine which sql
// upgrade file is the starting point for the next upgrade.
$v_major = '2';
$v_minor = '0';
$v_patch = '0';
$v_tag   = ''; // Tag should be empty for production releases

//use of this is a temporary work around for a bad descision in the past -tdm
$v_realpatch = $v_patch;

// Database version identifier, this is to be incremented whenever there
// is a database change in the course of development.  It is used
// internally to determine when a database upgrade is needed.
//
$v_database = '7';

// Access control version identifier, this is to be incremented whenever there
// is a access control change in the course of development.  It is used
// during installation to determine what the installed version of the access
// controls is (subsequently the acl_upgrade.php script then is used to
// upgrade and track this value)
//
$v_acl = 0;

?>

**Comment by teryhill** [comment]
I don't think we need the file it self it serves no real purpose. I think the updater that @naveen17797 did can handle the updates. I would like to keep the manual updates just as a backup in case something does not fire quite right with the automatic ones .

**Comment by tmccormi** [comment]
It is currently used to display the version on the login. but that is minimal and could be displayed as a "lated update installed" date (and source branch) or something.
